lib/config.js

```diff
@@ -1,44 +1,51 @@
-var defaults = require('lodash.defaults');
-var compact = require('lodash.compact');
-var map = require('lodash.map');
-var fs        = require('fs');
-var path      = require('path');
-var osHomedir = require('os-homedir');
+'use strict';
 
-var config = null;
+const defaults = require('lodash.defaults');
```

Comment:
Sure, not a priority, just wondering since the ones below are aligned. You were planning to do such formatting cleanups after the main conversion was complete (or would agree if they were made after this PR is merged. Just to be clear: would you be opposed to a code style cleanup PR afterwards?